modules/contentpool_normalization/src/EventSubscriber/ContentpoolNormalizationEventSubscriber.php

```diff
@@ -53,6 +53,31 @@ public function onAlterContentData(ReplicationContentDataAlterEvent $event) {
           $paragraph_data[$delta] = $entity->field_paragraphs->get($delta)->entity->toArray();
         }
         $normalized[$key]['field_data'][] = ['value' => json_encode($paragraph_data)];
+
+        if (!$entity->get('field_channel')->isEmpty()) {
+          /** @var \Drupal\taxonomy\Entity\Term $root_channel */
+          $channel = $entity->field_channel->entity;
+          if ($channel->hasField('field_remote_site') &&
+            !$channel->get('field_remote_site')->isEmpty()) {
+            $ancestors = \Drupal::service('entity_type.manager')
+              ->getStorage("taxonomy_term")
+              ->loadAllParents($channel->id());
+            if (!empty($ancestors)) {
```

Comment:
would be good to comment what you are doing here, it's not immediately obvious -> Find root element of the tree.